<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_Axis_Vth</class>
 <widget class="QWidget" name="widget_Axis_Vth">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>X_axis,Y_axis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/apprcc.qrc">
    <normaloff>:/APThrehold_icon.png</normaloff>:/APThrehold_icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_x">
     <item>
      <widget class="QLabel" name="label_X_axis">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;X_axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_x">
       <item>
        <widget class="QComboBox" name="comboBox_X_axis"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_X_axis_main"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_X_axis_sub"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_label"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Label&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_y">
     <item>
      <widget class="QLabel" name="label_Y_axis">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Y_axis&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_y">
       <item>
        <widget class="QComboBox" name="comboBox_Y_axis"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_Y_axis_main"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_Y_axis_sub"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_button">
     <item>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioButton_linear_fitting">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Linear fitting</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/apprcc.qrc"/>
 </resources>
 <connections/>
</ui>
